communications/README.md

```diff
@@ -0,0 +1,9 @@
+# Communications
+
+## Async written
+
+- If you want to discuss an idea (e.g. new or existing product feature, update a marketing page), comment on or create a new story/epic in Clubhouse. @-mention specific individuals and ask them specific questions. See [Product Development](../product-development/README.md).
+- Ensure your [Clubhouse notifications](https://help.clubhouse.io/hc/en-us/sections/201543416-Notifications) are properly configured. Respond to Tilters within a few days with an answer, or at least say you don't have time currently to do so.
+- If you want to discuss a process change, consider the smallest reasonable change and create a GitHub PR in this repo as a proposal. @-mention and assign specific individuals for review.
+- If you are having difficulty understanding each other over Clubhouse and/or GitHub, consider scheduling a meeting with a specific agenda to further discuss. But prefer starting with async communications first.
```

Comment:
I'd love something along the lines of "Afterwards, update Clubhouse/Github with any relevant output from the discussion, both to record the outcome and as a double-check on shared understanding."

Reply:
Thanks @landism. Definitely this is something I'd want to include, but probably for a more full-fledged meeting section in a later PR.